README.md

```diff
@@ -1382,6 +1382,22 @@ val handled: Stream[Int, Async] = original.handle(
 )
 ```
 
+To fuse one or more effects used to create and run a stream, you can use `unwrap` method.
+
+```scala
+import kyo.*
+
+case class Config(size:Int)
+
+val original: Stream[Int, Any] < Env[Config] = Env.get[Config].map(config => Stream.from(-9).take(config.size))
+
+val unwrapped: Stream[Int, Env[Config]] = original.unwrap
```

Comment:
You should probably either use the `Stream.unwrap` constructor or make some not that this method is in kyo-combinators.

Reply:
back to `kyo-prelude`, following: https://github.com/getkyo/kyo/pull/1244#discussion_r2135631160